vmcnet/train/default_config.py

```diff
@@ -53,6 +53,7 @@ def get_default_reload_config() -> ConfigDict:
             "config_relative_file_path": DEFAULT_CONFIG_FILE_NAME,
             "use_checkpoint_file": True,
             "checkpoint_relative_file_path": CHECKPOINT_FILE_NAME,
+            "model_only": False,
```

Comment:
Nit: here and elsewhere, I think the word "model" is a bit misleading since this is just the parameters and does not guarantee that the same model architecture is loaded. Can you update this to something like "params_only" or "model_params_only" and similarly update below your variable "reload_model_from_checkpoint"?

Reply:
It seems that parse_config_flags.parse_flags (runners.py line 583) actually loads the config from the reloaded run including the model config, in which case it does load the entire model. But then it might also load more config settings than that; is there a better name to reflect that?

Reply:
Well, that is true when `--reload.use_config_file=True`, but that's a separate toggle that's not related to your new `--reload.model_only` toggle. All the  `--reload.model_only` does is decide whether the checkpoint is used in its entirety, or whether it's used for only the parameters.

For concreteness, it's possible to set `use_config_file=False`, `use_checkpoint_file=True`, and `model_only=True`, which would result in the parameters but not the model architecture from being reloaded, which is an unexpected behavior if the flag is called `model_only` but would make perfect sense if it's called `params_only`.